src/main/webapp/js/common.js

```diff
@@ -534,6 +534,23 @@ function setStatusMessage(message, error) {
 }
 
 /**
+ * Append a message to the existing status message
+ * @param  message
+ * 
+ */
+function appendStatusMessage(message, error) {
+    if (message.trim() === '') {
+        return;
+    }
+    const currentContent = $(DIV_STATUS_MESSAGE).html();
```

Comment:
#4442 is failing because of this. @imouto1994, we can just change this to a var right?